x-pack/plugins/telemetry_collection_xpack/schema/xpack_plugins.json

```diff
@@ -5067,6 +5067,22 @@
             }
           }
         },
+        "top_traces": {
```

Comment:
is there a way to export this from `x-pack/plugins/apm/server/lib/apm_telemetry/schema.ts` ?

Reply:
@kpatticha do you know if we can export this?

Reply:
the [xpack_plugins.json](https://github.com/elastic/kibana/pull/166263/files#diff-3914e18700a9a8a7c4e4f763502be5f5cb91dc00251642bd6dd23470e46e03d8) is automatically updated by the schema. Running `node scripts/telemetry_check --fix` will do it for you. In reality, if you update the schema and not the json it will throw an error. 

is this what you meant by export @yngrdyn

Reply:
https://github.com/elastic/kibana/tree/main/packages/kbn-telemetry-tools#usage